fr/core-libraries/helpers/flash.rst

```diff
@@ -0,0 +1,46 @@
+Flash
+#####
+
+.. php:class:: FlashHelper(View $view, array $config = [])
+
+FlashHelper provides a way to render flash messages that were set in
+``$_SESSION`` by :doc:`FlashComponent </core-libraries/components/flash>`.
+:doc:`FlashComponent </core-libraries/components/flash>` and FlashHelper
+primarily use elements to render flash messages.  Flash elements are found under
+the ``app/View/Elements/Flash`` directory.  You'll notice that CakePHP's App
+template comes with two flash elements: ``success.ctp`` and ``error.ctp``.
+
```

Comment:
A short note that this helper and its sister component succeed the older ones would make it more clear for people having some knowledge about the old ones but haven't read the migration guide.